src/components/article/CommentCard.jsx

```diff
@@ -1,15 +1,68 @@
 import "./Article.css";
-import  dayjs  from 'dayjs'
-import advancedFormat from 'dayjs/plugin/advancedFormat'
+import dayjs from "dayjs";
+import advancedFormat from "dayjs/plugin/advancedFormat";
+import { useContext, useState } from "react";
+import { MdDelete } from "react-icons/md";
+import { UsernameContext } from "../../contexts/Username";
+import { deleteComment } from "../api";
+
+export default function CommentCard({ comment, handleViewComments }) {
+  dayjs.extend(advancedFormat);
+  const { username } = useContext(UsernameContext);
+  const [isDeleting, setIsDeleting] = useState(false);
+  const [isDeleted, setIsDeleted] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  function handleDeleteComment() {
+    setIsDeleting(true);
+    deleteComment(comment.comment_id)
+      .then(() => {
+        setIsDeleting(false);
+        setIsDeleted(true);
+        handleViewComments();
```

Comment:
You're reloading the comments from the API after every comment is added. On the upside it would mean you see comments others have used without refreshing. However it does negate the benefit of updating state and it efficiently just re-rendering components as necessary